persistence/src/main/java/org/apache/pekko/persistence/serialization/MessageFormats.java

```diff
@@ -1616,11 +1616,11 @@ public Builder clearSender() {
       }
       /**
        * <pre>
-       * optional int32 redeliveries = 6; // Removed in 2.4
-       * repeated string confirms = 7; // Removed in 2.4
-       * optional bool confirmable = 8;  // Removed in 2.4
-       * optional DeliveredMessage confirmMessage = 9; // Removed in 2.4
-       * optional string confirmTarget = 10; // Removed in 2.4
+       * optional int32 redeliveries = 6; // Removed in Akka 2.4
```

Comment:
We can probably delete these comments entirely

Reply:
I've removed those `<pre>` blocks now